src/lib/Libattr/master_job_attr_def.xml

```diff
@@ -2077,6 +2077,23 @@
          <ECL>verify_value_zero_or_positive</ECL>
       </member_verify_function>
    </attributes>
+   <attributes>
+      <member_index>JOB_ATR_server_inst_id</member_index>
+      <member_name>ATTR_server_inst_id</member_name>
+      <member_at_decode>decode_str</member_at_decode>
+      <member_at_encode>encode_str</member_at_encode>
+      <member_at_set>set_str</member_at_set>
+      <member_at_comp>comp_str</member_at_comp>
+      <member_at_free>free_str</member_at_free>
+      <member_at_action>NULL_FUNC</member_at_action>
+      <member_at_flags>PRIV_READ | ATR_DFLAG_SSET | ATR_DFLAG_NOSAVM</member_at_flags>
```

Comment:
Why 'ATR_DFLAG_NOSAVM' ? if you retrieve a job from db, won't this information be useful ?

Reply:
Yes, there is a possibility that the server instance name can be changed before it starts. This is the reason I have marked this attribute as ATR_DFLAG_NOSAVM. This is my understanding when I looked at the following code(pbsd_init_job function) with respect to the single server.

We are actually freeing the attribute JOB_ATR_at_server and every time populating its value. So Ideally this could have also been marked as ATR_DFLAG_NOSAVM.


```
	/* update at_server attribute in case name changed */
	free_jattr(pjob, JOB_ATR_at_server);
	set_jattr_generic(pjob, JOB_ATR_at_server, server_name, NULL, SET)
```
@subhasisb, please let us know your opinion on this.